### ISSUE TITLE
Confused by commits

### ISSUE THREAD

**Issue by JustinJaniszewski** [ubsuny/23-Homework3G3#62]
@s4il3sh I don't understand your latest pull request. You are rewriting over the code file with the graph, putting the code in documentation files. I don't understand it looks very messy. 

@WildJimmy please read carfully before merging it

**Comment by s4il3sh** [comment]
yes.. I've added graph to verify hooke's law. Also, I am following homework protocol to separate algorithm and created our own module.

**Comment by s4il3sh** [comment]
Inside documentation, I have added our code and output of pycodestyle as mentioned in our homework.

**Comment by JustinJaniszewski** [comment]
Alright, I'll take your word for it.
